litmus-portal/frontend/src/components/FinishModal/index.tsx

```diff
@@ -2,31 +2,24 @@ import Button from '@material-ui/core/Button';
 import Modal from '@material-ui/core/Modal';
 import React from 'react';
 import useStyles from './styles';
-
-/* Icon function is used for finish modal to show mark */
-function Icon() {
-  const classes = useStyles();
-
-  return <img src="icons/finish.png" className={classes.mark} alt="mark" />;
-}
+import ButtonFilled from '../Button/ButtonFilled';
 
 const FinishModal = () => {
   const classes = useStyles();
 
   const [open, setOpen] = React.useState(false);
-
+  
   const handleOpen = () => {
     setOpen(true);
   };
 
   const handleClose = () => {
     setOpen(false);
   };
-
-  /* Body part for modal */
+  
   const body = (
     <div className={classes.rootContainer}>
-      <Icon />
+      <img src="icons/finish.png" className={classes.mark} alt="mark" />;
       <div className={classes.heading}>
```

Comment:
.svg is preferred

Reply:
Thank you @Saranya-jena I will look into this.